topics/coverage/index.md

```diff
@@ -0,0 +1,10 @@
+---
+display_name: Coverage
```

Comment:
Just my opinion, but I think `Code coverage` should be the default topic. What do you think?

Reply:
I actually thought about that, too, but refrained from doing so as there is "code coverage" on one hand and "test coverage" on the other. What do you think about that?

Reply:
`Test coverage` might be a little bit better, but I'm okay with either one. I just think `Coverage` on its own is too generic to be clear out of context.

Reply:
I think this is most similar to the `react` vs `reactjs` example around aliases: https://github.com/github/explore/blob/3da6134c57074effddbc61e644502dacb6afc595/docs/API.md?plain=1#L5

For this one, I'd recommend using `Test coverage` as the topic name, with `coverage` and `code-coverage` as aliases. What do you think? I'm open to `Code coverage` as the topic name as well, but I think the aliases should cover us either way.

Reply:
https://www.browserstack.com/guide/code-coverage-vs-test-coverage
https://www.lambdatest.com/blog/code-coverage-vs-test-coverage/

I don't think, this is comparable to your example. But after all, it's your decision 🤷🏻‍♂️

Reply:
As this is probably the closest we'll come to an agreement, I just say, I'll make the necessary changes.